acme/tests/jws_test.py

```diff
@@ -62,5 +64,33 @@ def test_jwk_serialize(self):
         self.assertEqual(jws.signature.combined.jwk, self.pubkey)
 
 
+class JWSPayloadRFC8555Compliant(unittest.TestCase):
```

Comment:
nit: Since these tests don't test the code in `acme.jws`, can we move the tests to `challenge_test.py` and `messages_test.py` respectively?

Reply:
Done. About the message, I use a real class instead of a mock class to be closer to real cases.